drivers/SmartThings/zigbee-thermostat/profiles/base-thermostat-fanless.yml

```
@@ -12,6 +12,13 @@ components:
     version: 1
   - id: thermostatOperatingState
     version: 1
+    configuration:
```

```suggestion
    config:
```

@ctowns a similar change will need to be made to all the profiles in this PR

---

done